rs/ledger_suite/icrc1/ledger/tests/tests.rs

```diff
@@ -163,29 +167,31 @@ fn test_upgrade() {
     ic_ledger_suite_state_machine_tests::test_upgrade(ledger_wasm(), encode_init_args)
 }
 
-#[test]
-fn test_install_mainnet_ledger_then_upgrade_then_downgrade() {
-    ic_ledger_suite_state_machine_tests::test_install_upgrade_downgrade(
-        ledger_mainnet_wasm(),
-        encode_init_args,
-        ledger_wasm(),
-        encode_upgrade_args,
-        ledger_mainnet_wasm(),
-        encode_upgrade_args,
-    )
-}
+// It is not possible to downgrade to mainnet for now, disable the tests.
+// Similar testing is done by icrc1_test_downgrade_from_incompatible_version
```

Comment:
What do you mean by "similar testing"?

Reply:
Upgrading mainnet->current->current->mainnet etc. Since the test were not used I removed them, cc @mbjorkqvist